src/pages/FolderPage.jsx

```diff
@@ -1,16 +1,20 @@
-import AddLink from "../components/AddLink";
 import { getFolderLinks, getFolderCategory } from "../utils/api";
 import { useEffect, useState } from "react";
+import AddLink from "../components/AddLink";
 import Cards from "../components/Cards";
 import Search from "../components/Search";
 import Category from "../components/Category";
 import NoLink from "../components/NoLink";
-import ShowFolderName from "../components/ShowFolderName";
+import CategoryOption from "../components/CategoryOption";
+import Modals from "../components/modals/Modals";
+import { styled } from "styled-components";
 
 function FolderPage() {
   const [folderData, setFolderData] = useState([]);
   const [linkData, setLinkData] = useState([]);
   const [selectedFolder, setSelectedFolder] = useState("전체");
+  const [modalOn, setModalOn] = useState([false, null]);
```

Comment:
상태값을 배열보다는 객체로 관리하는 것이 더 좋아보입니다. 객체로 관리하게 되면 배열과 달리 순서와 무관하게 안정적으로 코드를 관리할 수 있습니다. 만약에 지금 보다 객체로 관리하는 요소가 늘어날 경우에 안정적입니다.

```suggestion
  const [modalOn, setModalOn] = useState({
    keyName1: false,
    keyName2: null,  
  });
```